doc/source/security/enabling-master-key.rst

```diff
@@ -0,0 +1,214 @@
+.. _enabling-master-key:
+
+==============================================================================
+Enabling Master Key Encryption
+==============================================================================
+
+.. _enabling-vault:
+
+Enabling the Keyring Vault
+---------------------------
+
+To enable Master key vault encryption, the user must have
+`SYSTEM_VARIABLES_ADMIN
+<https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_system-variables-admin>`__
+privileges.
+
+Add the following statements to my.cnf:
+
+.. code-block:: MySQL
+
+    [mysqld]
+    early-plugin-load="keyring_valut=keyring_valut.so"
+    loose-keyring_value_config="/home/mysql/keyring_vault.conf"
+
+Restart MySQL.
+
+To change the default value, use the following syntax:
+
+.. code-block:: guess
+
+   mysql> SET innodb_default_encryption_key_id = NEW_ID
+
+Here, **NEW_ID** is an unsigned 32-bit integer.
+
+
+Encrypting a General Tablespace
+-------------------------------------------
+
+.. _enabling-file-per-table:
+
+The InnoDB system tablespace is encrypted by using master key encryption. The
+server must be started with the ``--bootstrap`` option.
+
+If the variable :variable:`innodb_sys_tablespace_encrypt` is set to ON and the
```

Comment:
This is true when innodb_file_per_table is set to OFF. If file per table is on, you do not need system tablespace to be encrypted when creating new tables.

Reply:
Updated. Changed to OFF. Removed ``bootstrap`` references and added ``initialize``.